app_test.go

```diff
@@ -933,8 +947,8 @@ func TestApp_SetStdout(t *testing.T) {
 	var w bytes.Buffer
 
 	app := &App{
-		Name:   "test",
-		Writer: &w,
+		Name:      "test",
+		ErrWriter: &w,
```

Comment:
This function is specifically testing Stdout. Either rename this to TestApp_SetStdErr or clone this function